src/planner/components/BlockResource.tsx

```diff
@@ -60,7 +60,7 @@ export const BlockResource = (props: PlannerResourceProps) => {
             )}
             <foreignObject width={maxTextWidth} className="block-resource-text resource-name" y={padding} x={textX}>
                 <plannerRenderer.Outlet id={PlannerOutlet.ResourceTitle} context={props.actionContext}>
-                    <span>{props.name}</span>
+                    <span title={props.name}>{props.name}</span>
```

Comment:
We can use our own Tooltip inside foreignObject. We do that  e.g. here https://github.com/kapetacom/ui-web-plan-editor/blob/master/src/components/PlannerBlockWarningTag.tsx#L51